kairos-cli/tests/cli_tests.rs

```diff
@@ -0,0 +1,98 @@
+use assert_cmd::Command;
+use std::path::PathBuf;
+
+// Helper function to get the path to a fixture file
+fn fixture_path(relative_path: &str) -> PathBuf {
+    let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    path.extend(["tests", "fixtures", relative_path].iter());
+    path
+}
+
+#[test]
+fn deposit_successful_with_ed25519() {
+    let secret_key_path = fixture_path("ed25519/secret_key.pem");
+
+    let mut cmd = Command::cargo_bin("kairos-cli").unwrap();
+    cmd.arg("deposit")
+        .arg("--amount")
+        .arg("123")
+        .arg("--private-key")
+        .arg(secret_key_path);
```

Comment:
there is also `cmd.args(["deposit", "--amount", "123", "--private-key", secret_key_path])`

Reply:
This would make code more complex, as `args()` works only with `&str`, so passing `secret_key_path` requires additional work to convert it from `&Path` into `&str` (+handle error).